x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/route_resolver.ts

```diff
@@ -71,13 +70,7 @@ export async function resolver(
     layerIndex = undefined;
   }
 
-  const jobCreator = new QuickJobCreator(
-    getDataViews(),
-    getUiSettings(),
-    getTimefilter(),
-    getShare(),
-    ml
-  );
+  const jobCreator = new QuickJobCreator(getUiSettings(), getTimefilter(), getShare(), ml);
```

Comment:
this is the only place `getLens` can be used to supply `QuickJobCreator` with the lens start contract from within ML.

Reply:
got it, done